File: android/app/src/main/java/com/appxstudios/festivalconnection/models/WalletTransaction.kt
package com.appxstudios.festivalconnection.models

data class WalletTransaction(
    val id: String,
    val amountSat: Long,
    var amountUSD: Double,
    val direction: String, // "sent" or "received"
    val timestamp: Long,
    var description: String,
    var paymentHash: String,
    var status: String,
    var fees: Long = 0
)

data class WalletAlert(
    val amountSat: Long,
    val amountUSD: Double,
    val description: String,
    val timestamp: Long
)

File: ios/FestivalConnection/Models/WalletTransaction.swift
import Foundation

struct WalletTransaction: Identifiable, Equatable {
    let id: String
    let amountSat: UInt64
    var amountUSD: Double
    let direction: String // "sent" or "received"
    let timestamp: Date
    var description: String
    var paymentHash: String
    var status: String
    var fees: UInt64 = 0
}

struct WalletAlert: Identifiable {
    let id = UUID()
    let amountSat: UInt64
    let amountUSD: Double
    let description: String
    let timestamp: Date
}
